constants.py

```diff
@@ -1,4 +1,4 @@
 COLLECTIVE_DB = "07752fd5ba8e44c7b8e48bfee50f0545?"
 DATE_FMT = "%d %m %Y %H:%M"
 
-NOTION_NOTIFICATION_CHANNEL = 1126132159158173726
```

Comment:
Why did you change from the notion-hookz discord chan to the testing chan?

This probably should be reverted to the notion-hookz chan.

Reply:
Changed to testing chan while I was experimenting with the aggregate changes. 

Apologies on the error handling / logging, I made changes but when I reset a prior commit I did not add them back.

Reply:
@Sarahtonein Can you fix it?
You can add the error handling and logging later in a subsequent commit/branch/PR if you want

Reply:
@Balugrizzly ,

I've changed the channel back to Notion Hooks. Will do the error handling in a subsequent commit